src/index.js

```diff
@@ -49,6 +50,16 @@ function tileReduce(options) {
   return ee;
 }
 
-function bboxTiles(bbox, zoom) {
-  return tilecover.tiles(bboxPolygon(bbox).geometry, {min_zoom: zoom, max_zoom: zoom});
+function parseToTiles(options, zoom) {
+  var poly;
+
+  if (options.tiles) return options.tiles;
+
+  if (options.bbox) {
+    poly = bboxPolygon(options.bbox);
+  } else if (options.geojson) {
+    poly = options.geojson;
+  }
+
+  return tilecover.tiles(normalize(poly).features[0].geometry, {min_zoom: zoom, max_zoom: zoom});
```

Comment:
IMO normalize is kind of nice because it'll let us do features or geometries, or just in case somebody passes in a featurecollection or something of that sort. I'm kind of in favor of being a little forgiving with it